lms/djangoapps/instructor/views/api.py

```diff
@@ -2708,7 +2704,7 @@ def start_certificate_regeneration(request, course_id):
         )
 
     # Check if the selected statuses are allowed
-    allowed_statuses = GeneratedCertificate.get_unique_statuses(course_key=course_key, flat=True)
+    allowed_statuses = [CertificateStatuses.downloadable, CertificateStatuses.error, CertificateStatuses.notpassing]
```

Comment:
@stvstnfrd previously `allowed_statuses` were dynamically retrieved from `GeneratedCertificates`.  They were all unique values of `status` column of `GeneratedCertificate` for the current course.